Packs/Siemens_SiPass/README.md

```diff
@@ -0,0 +1,33 @@
+# Siemens SiPass
+
+<~XSIAM>
+
+This pack includes Cortex XSIAM content.
+
+## Configuration on Server Side
+The log forwarding of Siemens SiPass is configured using a [file collection][here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Activate-the-Files-and-Folders-Collector).
+
+**Importent Notes**
+* Details of the folder path on the network share containing the files that you want to monitor and upload to Cortex XSIAM.
+* Settings related to the list of files to monitor and upload to Cortex XSIAM, where the log format is either Raw (default), JSON, CSV, TSV, PSV, CEF, LEEF, Corelight, or Cisco.
```

Comment:
I don't understand these two bullets. Not sure what information you are trying to give.

Reply:
This are crucial steps of file collection, as mentioned in the file collection article 
![image](https://github.com/demisto/content/assets/147984773/63015b8e-9e4d-4086-a527-3391e8acf05c)